build-system/tasks/helpers.js

```diff
@@ -551,21 +552,11 @@ async function minify(code, map, {mangle} = {mangle: false}) {
     },
     sourceMap: {content: map},
     module: !!argv.esm,
+    nameCache,
   };
-
-  // Enabling property mangling requires disabling two other optimization.
-  // - Should not mangle quoted properties (often used for cross-binary purposes)
-  // - Should not convert computed properties into regular property definition
-  if (mangle) {
-    // eslint-disable-next-line google-camelcase/google-camelcase
-    terserOptions.mangle.properties = {keep_quoted: 'strict', regex: '_$'};
-    terserOptions.nameCache = nameCache;
-
-    // TODO: uncomment once terser bugs related to these are fixed
-    // https://github.com/terser/terser/pull/1058
-    terserOptions.compress.computed_props = false; // eslint-disable-line google-camelcase/google-camelcase
-    terserOptions.compress.properties = false;
-  }
+  // TS complains if defined inline, since it sees type `string` but needs type ("strict" | boolean).
+  // eslint-disable-next-line google-camelcase/google-camelcase
+  terserOptions.mangle.properties.keep_quoted = 'strict';
```

Comment:
My typescript doesn't seem to complain.

Reply:
![Screen Shot 2021-10-01 at 2 39 43 PM](https://user-images.githubusercontent.com/4656974/135670856-97cff5b5-dcdc-44ac-8232-b09276a38109.png)